### ISSUE TITLE
Fava terminal created regardless of runFavaOnActivate

### ISSUE THREAD

**Issue by fancl20** [Lencerf/vscode-beancount#77]
Fava terminal is created in the constructor of FavaManager:

```ts
  constructor(extension: Extension) {
    this._terminal = vscode.window.createTerminal('Fava');
    this._terminalClosed = false;
    this.extension = extension;
  }
```

IIUC, this will create a Fava terminal even runFavaOnActivate is set to false. Maybe we can set the terminal to close by default?

**Comment by Lencerf** [comment]
fixed in f8af0cc7b27df29db27fa00bcde85ab136f25445

**Comment by fancl20** [comment]
Verified. Thanks!
